### PR TITLE
feat(3): added homepage

### PR DESCRIPTION
**Pull request by taniasasaran**

### CODE REVIEW

File: lib/chat_app_web/live/home.ex
Comment (Davis856) [inline lib/chat_app_web/live/home.ex:1]: I would change the module name to HomeLive. The reason for that is that it's considered standard procedure, as we also have LiveComponents which are also placed in the "live" directory, as "NameComponent".

File: assets/css/app.css [hunk 1]
```diff
@@ -3,3 +3,68 @@
...
+
+.draw#jos{
+    top: 50px;
+    left: -500px;
+}
+
+.draw#sus{
+    width: 300px;
+    height: 300px;
+    transform: rotate(180deg);;
```
Comment (Davis856) [inline assets/css/app.css]: Extra semicolon here, small mistake but worth noticing.

File: lib/chat_app_web/live/home_live.ex [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    socket = assign(socket, :rooms, rooms)
+    {:ok, socket}
+  end
+
+  def render(assigns) do
+    ~H"""
+    <.header class="hd">Public Rooms</.header>
+    <.button id="newroom" class="hd">New Room</.button>
+
+    <.table id="rooms" rows={@rooms}>
```
Comment (danfilip) [inline lib/chat_app_web/live/home_live.ex:15]: aici @rooms nu trebuie luat din assigns?

File: lib/chat_app_web/components/core_components.ex [hunk 3]
```diff
@@ -454,25 +454,25 @@ defmodule ChatAppWeb.CoreComponents do
...
+                <span class={["relative text-zinc-900 group-hover:bg-forestgreen group-hover:text-zinc-50", i == 0 && "font-semibold"]}>
                   <%= render_slot(col, @row_item.(row)) %>
                 </span>
               </div>
             </td>
             <td :if={@action != []} class="relative w-14 p-0">
               <div class="relative whitespace-nowrap py-4 text-right text-sm font-medium">
-                <span class="absolute -inset-y-px -right-4 left-0 group-hover:bg-zinc-50 sm:rounded-r-xl" />
+                <span class="absolute -inset-y-px -right-4 left-0 sm:rounded-r-xl " />
                 <span
```
Comment (salajandan) [inline lib/chat_app_web/components/core_components.ex:473]: i think you can remove the extra space

**Review by salajandan — APPROVED** [review]

**Review by danfilip — CHANGES_REQUESTED** [review]

File: lib/chat_app_web/live/home_live.ex (hunk 2 above)
Comment (salajandan) [reply lib/chat_app_web/live/home_live.ex:15]: @danfilip acel "@" in HEEX templates presupune ca variabila rooms e luata din assigns.

**Review by danfilip — APPROVED** [review]

**Pull request merged**